cluster/gce/util.sh

```diff
@@ -660,7 +660,7 @@ function create-network() {
       --project "${PROJECT}" \
       --network "${NETWORK}" \
       --source-ranges "10.0.0.0/8" \
-      --allow "tcp:1-65535,udp:1-65535,icmp" &
+      --allow "tcp:1-2379,tcp:2382-65535,udp:1-65535,icmp" &
```

Comment:
There is a slight semantic different: before we were allowing all the nodes, now only minions and masters.

However, this sounds reasonable. Done